src/components/Workspace/utils.ts

```diff
@@ -117,18 +13,9 @@ export const getNamespaceUsingWorspaceFromQueryCache = async (
 export const getWorkspaceUsingNamespaceFromQueryCache = async (
```

Comment:
I believe the `async` keyword here is no longer required since we are using `Promise.resolve()`.

```suggestion
export const getWorkspaceUsingNamespaceFromQueryCache = (
```